src/components/accessories/billTable/types.ts

```diff
@@ -0,0 +1 @@
+export const totalRow = 10;
```

Comment:
if this is the only thing that this file contains, it should be renamed const.ts

Reply:
OK, I removed it.